packages/atomic/src/components.d.ts

```diff
@@ -1636,7 +1671,7 @@ declare namespace LocalJSX {
     }
     interface AtomicResultIcon {
         /**
-          * Specifies the icon to display from the list of available icons.  By default, this will parse the `objecttype` and `filetype` fields to find a matching icon. If none are available, it will use the `custom` icon.
+          * Specifies the icon to display, either from the list of available icons or a direct link.  By default, this will parse the `objecttype` and `filetype` fields to find a matching icon. If none are available, it will use the `custom` icon.
```

Comment:
```suggestion
          * Specifies the icon to display, either from the list of available icons or a direct link. By default, this will parse the `objecttype` and `filetype` fields to find a matching icon. If none are available, it will use the `custom` icon.
```